views/my_packages.package.dt

```diff
@@ -19,6 +19,21 @@ block body
 				|= e.get!string
 				br
 
+	h2 General
+
+	.inputForm
+		form(method="POST", enctype="multipart/form-data", action="#{req.rootDir}my_packages/#{packageName}/set_logo")
+			h3 Logo
+			- string logoPath = "/packages/" ~ packageName ~ "/logo";
+			img.packageLogo(src=logoPath)
+			p.logoError.error
+			label
+				| Select new Logo (at most 1 MiB, optimal size 512x512)
```

Comment:
Nit: typically `max.` is used for this (or not displayed at all except for the error case)